examples/node8-yarn/docker-compose.override.yml

```diff
@@ -0,0 +1,24 @@
+version: '3.3'
+
+services:
+
+  api:
+    # Assumes there's a development mode configured via a package.json "watch"
+    # script. This replaces the "yarn start" run by default.
+    command: ["yarn", "run", "watch"]
+    environment:
+      DOCKER_ENV: ${DOCKER_ENV:-local}
+      NODE_ENV: ${NODE_ENV:-development}
+    # @todo revisit after https://github.com/phase2/rig/issues/135 is resolved.
+    user: root
+    volumes:
+      # Use Outrigger Filesync to maximize performance of build times.
+      # If you have issues with filesync and do not need file watches, please
+      # test the performance by replacing "dullahan-sync" with a period (.:/code)
+      - dullahan-sync:/code
+      # Persist tool caches. This covers yarn and npm.
+      - /data/dullahan/cache:/usr/local/share/.cache
+
+volumes:
+  dullahan-sync:
```

Comment:
Some copypasta in this file. Replace dullahan with node8-yarn-example or similar.